docs/knowledgebase/learn-substrate/off-chain-features.md

```diff
@@ -104,3 +106,4 @@ refer to the following sections in Substrate Recipes:
 - <span id="example-txs">[Submit signed and unsigned transactions from off-chain workers back on-chain](https://substrate.dev/recipes/off-chain-workers/transactions.html)</span>
 - <span id="example-http">[Fetch external data using HTTP requests and parse JSON responses](https://substrate.dev/recipes/off-chain-workers/http-json.html)</span>
 - <span id="example-off-chain-storage">[Store result in off-chain worker local storage](https://substrate.dev/recipes/off-chain-workers/storage.html)</span>
+- <span id="example-off-chain-indexing">[Example of off-chain indexing](https://substrate.dev/recipes/off-chain-workers/indexing.html)
```

Comment:
this is a 404 for me at the moment. Do you mean https://substrate.dev/recipes/off-chain-workers/ ?

Reply:
Perhaps you meant this? -> https://substrate.dev/docs/en/knowledgebase/learn-substrate/off-chain-features#off-chain-indexing

Reply:
i see. This is due to recipes is not deployed in gh-pages, which is pending on https://github.com/substrate-developer-hub/recipes/issues/438

Reply:
This issue resolved, and Recipes is deployed. Indexing page can be accessed [here](https://substrate.dev/recipes/off-chain-workers/indexing.html).